tests/schemas/test_ticket_schema.py

```diff
@@ -13,3 +13,19 @@ def test_valid_payload(self, create_tenant, create_admin_user):
 
         no_validation_errors = {}
         assert no_validation_errors == TicketSchema().validate(valid_payload)
+
+    def test_valid_payload_with_note(
+        self, create_tenant, create_admin_user, create_note
+    ):
+
+        author_id = create_admin_user().id
+        valid_payload = {
+            "tenant_id": create_tenant().id,
+            "author_id": author_id,
+            "note": {**create_note().json(), "user_id": author_id},
+        }
+
+        del valid_payload["note"]["user"]
```

Comment:
This is a bit confusing. I'm not sure what the intent is here. Also your passing in a created note json fields to the valid_payload. 

The valid payload should represent what we would receive from the frontend, and looking at this I'm not quite sure what that is. I should be able to read the test and have a good understanding of the behavior of the application. Here, I'm not so sure what's going on with note.